modules/services/wob.nix

```diff
@@ -37,6 +37,19 @@ in {
 
     systemd = mkEnableOption "systemd service and socket for wob"
       // mkOption { default = true; };
+
+    systemdTarget = mkOption {
```

Comment:
`systemd` is sadly already used to enable the service above, so using `systemdTarget` instead to avoid breaking changes.